sbi/inference/base.py

```diff
@@ -79,20 +76,13 @@ def __init__(
         self._show_progressbar = show_progressbar
         self._show_round_summary = show_round_summary
 
-        if num_workers > 1:
-            self._batched_simulator = lambda theta: simulate_in_batches_mp(
-                self._simulator,
-                theta,
-                simulation_batch_size,
-                num_workers,
-                worker_batch_size,
-                self._show_progressbar,
-                logging_level=logging_level,
-            )
-        else:
-            self._batched_simulator = lambda theta: simulate_in_batches(
-                self._simulator, theta, simulation_batch_size, self._show_progressbar
-            )
+        self._batched_simulator = lambda theta: simulate_in_batches(
+            self._simulator,
+            theta,
+            simulation_batch_size,
+            num_workers,
+            self._show_progressbar,
+        )
```

Comment:
wait, what? I thought we decided not to get rid of `simulate_in_batches()` without joblib just yet?

Reply:
Ah, you moved it. Nice!